Sources/SwiftKafka/Configuration/KafkaProducerConfiguration.swift

```diff
@@ -36,6 +36,8 @@ public struct KafkaProducerConfiguration {
     /// When set to true, the producer will ensure that messages are successfully produced exactly once and in the original produce order. The following configuration properties are adjusted automatically (if not modified by the user) when idempotence is enabled: max.in.flight.requests.per.connection=5 (must be less than or equal to 5), retries=INT32_MAX (must be greater than 0), acks=all, queuing.strategy=fifo. Producer instantation will fail if user-supplied configuration is incompatible.
     /// Default: `false`
     public var enableIdempotence: Bool = false
+    
+    public var transactionalId: String?
```

Comment:
We need some comments on this and what it does. If it only makes sense for the transaction producer we should not expose it all in this configuration and maybe have to introduce a new `KafkaTransactionalProducerConfiguration`

Reply:
Do you think of some wrapper around producer configuration like the following?
```
struct KafkaTransactionalProducerConfiguration {
    var producerConfiguration: KafkaTransactionalProducerConfiguration
    public var transactionalId: String
}
```

Reply:
I am not sure that works. Presumably there are some configurations that don't apply to a transactional producer. We probably should introduce a new configuration and just duplicate stuff like we did for the consumer and producer.